bindings/python/tests/pire_test.py

```diff
@@ -0,0 +1,80 @@
+import pire
+import pytest
```

Comment:
Order of imports:

  * group in 3 groups: system, 3d party, local;
  * sort alphabetically (case-insensitive) in each group.